projects/plugins/jetpack/_inc/lib/class.core-rest-api-endpoints.php

```diff
@@ -1641,7 +1641,7 @@ public static function rewind_data() {
 
 		$body   = wp_remote_retrieve_body( $response );
 		$result = json_decode( $body );
-		set_transient( 'jetpack_rewind_state', $result, 30 * MINUTE_IN_SECONDS );
+		set_transient( 'jetpack_rewind_state', $result, 1 * DAY_IN_SECONDS );
```

Comment:
Since transients seem to get refreshed when plans are canceled, I'm not sure there is a need for them to expire so fast. It's possible they could live even longer (3 days, a week?) but I'm not sure if something else is affected.

Reply:
I'm wondering about the opposite here - what are the implications if the transient lasts longer but those capabilities (scan / backup) have been removed from a site? 🤔

Reply:
In case plans are removed from a site, transients get updated with status `unavailable` and `missing_plan` so that shouldn't cause problems. Or, at least that didn't happen in my tests.